lib/responder.js

```diff
@@ -1119,6 +1119,11 @@ function resolvedPromise(operation, resolve) {
   }
 
   var data    = operation.data;
+  for(let i=0; i<data.length; i++){
+    if(data[i].format === 'text' && typeof data[i].value !== 'string'){
```

Comment:
For sanity purposes, include a null check on `data[i].value` as well as we don't want to call `toString()` on it if it's null.